.vsts-ci/windows.yml

```diff
@@ -69,6 +69,11 @@ stages:
       purpose: ElevatedPesterTests
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Reflection.Emit.dll; SideIndicator=<=}, @{InputObject=System.Threading.Timer.dll; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 127
127:         $diffs | Should -BeExactly $null
```